File: src/cpp/katalet.cpp
#include <iostream>
#include <string>

int main( int argc, char *argv[] )
{
    std::string kataName = argv[1];
    std::cout << "The katalet you have selected is " << kataName << std::endl;
    return 0;
}

//class Katalet
//
//{
//    public static void main (String[] args) {
//        String kataName = args[0];
//        System.out.format("The katalet you have selected is %s\n", kataName);
//    }
//}

File: src/cs/katalet.cs
public class Katalet
{
    public static int Main (string[] args)
    {
        if( args.Length == 0) {
            System.Console.WriteLine("Please enter a katalet name.");
            return 1;
        }

        string kataName = args[0];
        System.Console.WriteLine("The katalet you have selected is {0}", kataName);
        return 0;
    }
}
